test-app/app/components/disclosure/basic.hbs

```diff
@@ -0,0 +1,61 @@
+<div class='w-full px-4 py-4 pt-16'>
+  <div class='mx-auto w-full max-w-md bg-white shadow-md px-4 py-4 rounded-lg'>
+    <Disclosure as |disclosure|>
+      <span class='rounded-md shadow-sm'>
+        <disclosure.Button
+          @index={{1}}
```

Comment:
we do we have index here? A consumer should never need to manage the index, yeah?